packages/gatsby/src/db/common/query.ts

```diff
@@ -50,7 +50,7 @@ export function createDbQueriesFromObject(filter: object): Array<DbQuery> {
 }
 
 function createDbQueriesFromObjectNested(
-  filter: object,
+  filter: Record<string, any>,
```

Comment:
Can this be `Record<string, unknown>`

Reply:
Changing the `any` to `unknown` triggers cascading errors. Are you sure that's the right way?

Reply:
May be related to this:

> packages/gatsby/src/db/common/query.ts:63:57 - error TS2345: Argument of type 'unknown' is not assignable to parameter of type 'Record<string, unknown>'.


I hate that my editor throws different errors from `yarn typecheck` from whatever `yarn format` does. There should be one state :/